packages/yew/src/virtual_dom/vtag.rs

```diff
@@ -430,8 +431,28 @@ impl VTag {
             .insert(key, value.into_prop_value());
     }
 
-    /// Set event listeners on the [VTag]'s  [Element]
+    /// Add pending event listener on the [VTag]'s  [Element]
```

Comment:
Can you add a little note about what a pending listener is here? It isn't clear what pending listener is from just looking at this method.

Reply:
Yeah what is a "pending event listener"?

Reply:
A pending listener is a `Listener` instance pending for registration in the global event listener multiplexer. That's an internal implementation detail, that need not be exposed. This function is better named `add_listener()` and the other public function can be `set_listeners()`, as you have commited. 

`set_listener()` is a misnomer and not needed. I think it got in here via improper merge conflict resolution on my part.

Reply:
I still have not received an answer anywhere why would the average yew user need this?
To my understanding it just skips the `html!` macro?
Cant users that dislike that macro use something like `yew-vdom-gen` instead of hacking through yew internal api? (I get the irony that `yew-vdom-gen` uses same internal api)

Reply:
@bakape @voidpumpkin I renamed this function to `add_listener`.